lib/hammer_cli_foreman/compute_resource.rb

```diff
@@ -31,20 +31,19 @@ class InfoCommand < HammerCLIForeman::InfoCommand
 
       PROVIDER_SPECIFIC_FIELDS = {
         'ovirt' => [
-          Fields::Field.new(:label => _('UUID'), :path => ["compute_resource", "uuid"])
+          Fields::Field.new(:label => _('UUID'), :path => [:uuid])
```

Comment:
this does not work without also patching Foreman to include UUID in json output, please open a separate PR adding `:uuid` to https://github.com/theforeman/foreman/blob/develop/app/views/api/v2/compute_resources/ovirt.json.rabl#L1